delegation-toolkit/reference/api/smart-account.md

```diff
@@ -372,12 +372,12 @@ Creates a `MetaMaskSmartAccount` instance.
 | Name | Type                                                | Required                                                     | Description                                                                                                                                                                       |
 | ---- |-----------------------------------------------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `client` | `Client`                                            | Yes                                                          | Viem Client to retrieve smart account data.                                                                                                                                       |
-| `implementation` | `TImplementation`                                   | Yes                                                          | Implementation type for the smart account. Can be Hybrid or Multisig.                                                                                                             |
+| `implementation` | `TImplementation`                                   | Yes                                                          | Implementation type for the smart account. Can be Hybrid, Multisig, or Stateless7702.                                                                                                             |
```

Comment:
So we now support an additional smart account type?
This probably requires updates to the following?
- https://docs.metamask.io/delegation-toolkit/concepts/smart-accounts/#smart-account-implementation-types
- https://docs.metamask.io/delegation-toolkit/how-to/create-smart-account/configure-accounts-signers/

Reply:
@bgravenorst There is a thread about this in [Slack](https://consensys.slack.com/archives/C05LHCHB82G/p1750770462626549).

Reply:
Cool, that solution in Slack good